plugin/al/mayautils/AL/maya/utils/PluginTranslatorOptions.cpp

```diff
@@ -544,21 +560,54 @@ void PluginTranslatorOptions::generateIntGlobals(
 
 //----------------------------------------------------------------------------------------------------------------------
 void PluginTranslatorOptions::generateFloatGlobals(
-    const char* const prefix,
-    const MString&    niceName,
-    const MString&    optionName,
-    MString&          code,
-    float             value)
+    const char* const  prefix,
+    const MString&     niceName,
+    const MString&     optionName,
+    MString&           code,
+    float              value,
+    int                precision,
+    const std::string& controller,
```

Comment:
I would be nicer to pass as  MString, other parameters are passed using that type (except prefix for some reason). It gets converted to a MString in the function anwyay. Other wise the parameters would now take all three of: const char*, MString and std::string.

Reply:
Done.